catlin/pkg/validator/content_validator.go

```diff
@@ -42,6 +51,11 @@ func (v *ContentValidator) Validate() Result {
 	name := v.res.Name
 	resName := fmt.Sprintf("%s: %s - name: %q", kind, gvk.GroupVersion(), name)
 
+	cat, err := getCategories()
```

Comment:
catlin supports dir validation so for eg. when `task/buildah/*` is passed this will executed multiple time, so if we can pass the list from outside to `NewContentValidator` would save the http calls
wdyt?

Reply:
Updated !